specs/altair/sync-protocol.md

```diff
@@ -141,6 +128,50 @@ def get_safety_threshold(store: LightClientStore) -> uint64:
     ) // 2
 ```
 
+### `is_better_update`
+
+```python
+def is_better_update(new_update: LightClientUpdate, old_update: LightClientUpdate) -> bool:
```

Comment:
`is_better_update` makes the rules more clear 👍

Reply:
Also allows some degree of reuse in server side code (although the conditions are likely checked before computing the full update)